accepted/global-json-updates.md

```diff
@@ -0,0 +1,374 @@
+      * Otherwise, fail.
+    * `latestPatch`
+      * Even if the requested major/minor/feature band/patch is found, do not use it unless it matches rules below.
+      * Considering only the equal and higher patches in the major/minor/feature band, select the highest available patch.
+      * If there are no equal or higher patches in the major/minor/feature band, fail.
+    * `latestFeature`
+      * Even if the requested major/minor/feature band is found, do not use it unless it matches rules below.
+      * Considering only the equal and higher feature bands in the major/minor, select the highest available feature band and its highest patch.
+      * If there are no equal or higher feature bands in the major/minor, fail.
+    * `latestMinor`
+      * Even if the requested major/minor is found, do not use it unless it matches rules below.
+      * Considering only the equal and higher minor versions in the major version, select the highest available minor, its highest feature band and its highest patch.
+      * If there are no equal or higher minor versions in the major, fail.
+    * `latestMajor` (latest)
+      * Even if the requested major is found, do not use it unless it matches rules below.
+      * Considering only the equal and higher major versions, select the highest available major, its highest minor, its highest feature band and its highest patch.
+      * If there are no equal or higher major versions, fail.
+    * `disable`
+      * Do not roll forward. Only bind to specified version. This policy means the latest patches, including security patches will not be used.
+ 
+## Default
+
+### No `global.json`
+
+If there is no `global.json` or a version is not specified, the default is`latestMajor`, again to match today's behavior.
+
+### `global.json` containing a version number
+
+**Change** The default behavior is changed. We took this change because we believe the new behavior will cause no change or be an improvement. 
+
+The new behavior is `major` as described above. The previous behavior is `patch` as described above.
+
+This changes behavior in two scenarios:
+
+* Previously, if the specified patch version was present, it was used even if a higher patch number was installed. In discussing this with users, ignoring the updated patch was not the anticipated behavior. 
+* If there was no patch version of the specified major/minor/feature band, SDK resolution previously failed. SDK resolution will now succeed. 
+
+This will change behavior if the specified SDK is not present, but a higher version outside the current feature band is present. Previously SDK selection failed in this scenario, and now it will succeed.
+
+## Examples to illustrate roll-forward options
+
+The following table considers various SDK version combinations that may be available on the machine, and the outcome of the following `global.json`:
+
+```json
+{
+  "sdk": {
+    "version": "2.1.501",
+    "rollForward": [entry in table header]
```

Comment:
This syntax is confusing as the '[' and ']' signify a json dictionary. This would be my preference.
```suggestion
    "rollForward": "<entry in table header>"
```